corehq/apps/sms/migrations/0001_initial.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import corehq.apps.sms.models
+import dimagi.utils.couch.migration
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('smsforms', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='MessageLogOld',
```

Comment:
It's here due to automatically creating migrations using `./manage.py makemigrations` in django 1.7.